src/dotty/tools/dotc/config/ScalaSettings.scala

```diff
@@ -40,7 +40,7 @@ class ScalaSettings extends Settings.SettingGroup {
   val jvmargs = PrefixSetting("-J<flag>", "-J", "Pass <flag> directly to the runtime system.")
   val defines = PrefixSetting("-Dproperty=value", "-D", "Pass -Dproperty=value directly to the runtime system.")
   val toolcp = PathSetting("-toolcp", "Add to the runner classpath.", "")
-  val nobootcp = BooleanSetting("-nobootcp", "Do not use the boot classpath for the scala jars.")
```

Comment:
I do not think we should remain options inherited from scalac, unless they have different meaning.